lib/rspec/core/notifications.rb

```diff
@@ -359,7 +361,11 @@ def formatted_load_time
       # @return [String] The summary information fully formatted in the way that
       #   RSpec's built-in formatters emit.
       def fully_formatted(colorizer=::RSpec::Core::Formatters::ConsoleCodes)
-        formatted = "\nFinished in #{formatted_duration} " \
+        formatted = ""
+        if syntax_highlighting_unavailable
+          formatted << "\nSyntax highlighting of failure snippets unavailable -- install the coderay gem to enable it.\n"
```

Comment:
I'm not going to bring in coderay gem on all of my projects just for this feature, so I'm always going to get this message. That's kind of annoying I think.

It's a hard balance between advertising and getting out of the way!

We'd have to add a config flag for it, which seems possibly overkill.

Maybe leave it as is and I'll send you a PR if it's really getting to me ;)